README.md

```diff
@@ -7,118 +7,28 @@ RubiX is a light-weight data caching framework that can be used by Big-Data engi
 RubiX can be extended to support any engine that accesses data in cloud stores using Hadoop FileSystem interface via plugins. 
 Using the same plugins, RubiX can also be extended to be used with any cloud store
 
-### Usecase
+Check the [User and Developer manual]
+## Usecase
 
 RubiX provides disk or in-memory caching of data, which would otherwise be accessed over network when it resides in cloud store,
```

Comment:
should we say in-memory ? I know we use page cache but in terms of "in-memory" people might think something else